openc3-traefik/traefik-ssl.yaml

```diff
@@ -1,23 +1,25 @@
 ---
 tls:
-  certificates:
-    - certFile: "/etc/traefik/cert.crt"
-      keyFile: "/etc/traefik/cert.key"
+  stores:
+    default:
+      defaultCertificate:
```

Comment:
Did this change in traefik?

Reply:
Works better doing it this way, less picky.